src/components/molecules/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import styles from './AnnouncementBanner.module.css'
+import Markdown from '../atoms/Markdown'
+import { useWeb3 } from '../../providers/Web3'
+import {
+  addCustomNetwork,
+  addOceanToWallet,
+  NetworkObject
+} from '../../utils/web3'
+import { getOceanConfig } from '../../utils/ocean'
+import { getProviderInfo, IProviderInfo } from 'web3modal'
+import { useOcean } from '../../providers/Ocean'
+import { useSiteMetadata } from '../../hooks/useSiteMetadata'
+import Button from '../atoms/Button'
+
+export interface AnnouncementAction {
+  name: string
+  style?: string
+  handleAction: () => void
+}
+
+export default function AnnouncementBanner(): ReactElement {
+  const { web3Provider, networkId } = useWeb3()
+  // const [providerInfo, setProviderInfo] = useState<IProviderInfo>()
+  const { config, connect } = useOcean()
+  const { announcement } = useSiteMetadata()
+
+  const network: NetworkObject = {
+    chainId: 137,
+    name: 'Matic Network',
+    urlList: [
+      'https://rpc-mainnet.matic.network',
+      'https://rpc-mainnet.maticvigil.com/'
+    ]
+  }
+  const [text, setText] = useState<string>(announcement.main)
+  const [action, setAction] = useState<AnnouncementAction>()
+  const addCustomNetworkAction = {
+    name: 'Add custom network',
+    handleAction: () => addCustomNetwork(web3Provider, network)
+  }
+  const addCustomTokenAction = {
```

Comment:
sorry, misunderstanding. What I meant with the _Get mOCEAN_ action was to link that to the bridge tutorial, not add as custom token. 

Or, another more simple way might be to just include the sentence _Use the Polygon Bridge to get mOCEAN_ in the announcement message, and link that sentence to the bridge tutorial. Then remove the button for wallet + polygon condition

Reply:
ok, yeah I got this wrong